src/main/java/seedu/address/model/dayData/CustomQueue.java

```diff
@@ -243,30 +243,34 @@ public int hashCode() {
     }
 
     /** Returns true if {@code dayDatas} table constraints are enforced */
-    private boolean tableConstraintsAreEnforced(List<DayData> dayDatas) {
-        int count = 0;
+    public static boolean tableConstraintsAreEnforced(List<DayData> dayDatas) {
```

Comment:
Mmm, given that statistics is independent of user input, i feel like if the tableConstraints aren't meant, we shouldn't throw an error? We should just do an internal fix like reset data or recreate table based on remaining valid data